index.html

```diff
@@ -6,6 +6,22 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="/favicon.ico" />
     <title>Cuttle</title>
+
+    
+    <meta property="og:url" content="https://www.cuttle.cards/" />
+    <meta property="og:title" content="Cuttle Cards Online" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="/public/img/cuttle_meta_img.jpg/" />
+    <meta property="og:description"
+        content="Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free!" />
```

Comment:
This should likely be shared somewhere or possibly set in vite as template vars.

Reply:
@seriouslysean from what I can tell, vue doesn't natively support dynamic `<meta>` tags. It looks like if we want to make it programatic we could use the [unhead lib](https://unhead.harlanzw.com/guide/getting-started/setup) which seems to be the closest thing to a standard approach I can find.

That said, the main reason for making this programatic is basically to be able to reuse the content values for the og (facebook) and twitter properties, right? Curious whether it's worth a dependency to manage that complexity. Penny for your thoughts

Reply:
You could also use something like https://vue-meta.nuxtjs.org/

I think the benefit is that you’ll be able to easily swap based on the page type without too much overhead. So rules, homepage, etc could all have their own content.

this is particularly nice if you’re planning on building this out and making localized pages behind a context path like /es or /fr, etc.

Reply:
I think that makes sense and could reasonably be a follow up issue. First step is just getting the static metadata in there.

When we have a clearer vision for what page-specific metadata should be, we can implement `useHead()` to make it dynamic

Reply:
Twitter recommends a char limit of 200 for the content, so let's reword this
```suggestion
        content="Cuttle is a battle card game played with a standard 52-card deck of cards. The strategic nuance of trading card games, with the elegant balance of a standard deck--and you can play it for free!" />
```